mibidata/tests/test_mibi_image.py

```diff
@@ -40,6 +55,8 @@ def setUp(self):
         warnings.filterwarnings(
             'ignore',
             message='Anti-aliasing will be enabled by default.*')
+        self.maxDiff = None
```

Comment:
What is `maxDiff` for? I don't see it used anywhere.

Reply:
It's for py.test to print full output. It's useful for debugging and it has no effect on the user. It was already defined in the tests for `tiff` so I activated it here as well.